stdlib/2/sets.pyi

```diff
@@ -0,0 +1,60 @@
+# Stubs for sets (Python 2)
+from typing import Any, Callable, Hashable, Iterable, Iterator, Mapping, Optional, TypeVar, Union
+
+_T = TypeVar('_T')
+Setlike = Union['BaseSet', Iterable]
```

Comment:
Needs to be prefixed with `_` to indicate that it's private. I think it should also be a generic type alias `_Setlike = Union[BaseSet[_T], Iterable[_T]]`.

Also, in stubs you don't need to quote names that aren't defined yet.